pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -1710,7 +1710,7 @@ public void analyzeSubscriptionBacklog(
             @PathParam("subName") String encodedSubName,
             @ApiParam(name = "position", value = "messageId to start the analysis")
             ResetCursorData position,
-            @ApiParam(value = "Is authentication required to perform this operation")
+            @ApiParam(value = "Whether leader broker redirected this call to this broker. For internal use.")
```

Comment:
```suggestion
            @ApiParam(value = "Whether leader broker redirects this call to this broker. For internal use.")
```
Do you mean this?

Reply:
No, I meant to use the past tense for redirect because the redirect happened in the past. Thanks for asking.